ssc/cmod_merchantplant.cpp

```diff
@@ -1333,7 +1333,9 @@ class cm_merchantplant : public compute_module
 			cf.at(CF_om_opt_fuel_2_expense,i) *= om_opt_fuel_2_usage;
 		}
 
-        std::vector<double> mp_energy_market_price = lookup("mp_energy_market_price")->arr_vector();
+        std::vector<double> mp_energy_market_price(8760*nyears, 0.0);
+        if (lookup("mp_energy_market_price"))
+            std::vector<double> mp_energy_market_price = lookup("mp_energy_market_price")->arr_vector();
```

Comment:
Re-declaration of mp_energy_market_price almost certainly is generating a compiler warning.

Reply:
Updated, thanks!